custom_components/tuya_local/devices/dry_pury_evo_dehumidifier.yaml

```diff
@@ -0,0 +1,160 @@
+name: Argo DRY PURY EVO WF dehumidifier
+products:
+  - id: oya6k3fqen43zyi7
+    name: Argo DRY PURY EVO WF 21
+primary_entity:
+  entity: humidifier
+  class: dehumidifier
+  dps:
+    - id: 1
+      name: switch
+      type: boolean
+    - id: 2
+      name: humidity
+      type: integer
+      range:
+        min: 30
+        max: 80
+      mapping:
+        - step: 5
+    - id: 5
+      name: mode
+      type: string
+      mapping:
+        - dps_val: dehumidify
+          value: comfort
+        - dps_val: drying
+          value: laundry
+          icon: "mdi:tshirt-crew-outline"
```

Comment:
If you define value specific icons here, you will need to define a default icon as well (most likely a higher priority (lower number) icon for off, and a lower priority (higher number) for on)